src/frameworks/audio_toolbox/audio_services.rs

```diff
@@ -33,4 +35,13 @@ fn AudioServicesGetProperty(
     }
 }
 
-pub const FUNCTIONS: FunctionExports = &[export_c_func!(AudioServicesGetProperty(_, _, _, _, _))];
+fn AudioServicesPlaySystemSound(_env: &mut Environment, in_system_sound_id: SystemSoundID) {
+    assert_eq!(in_system_sound_id, kSystemSoundID_Vibrate)
+    // TODO: vibrate on devices supporting it
```

Comment:
Could you make this TODO into a `log!` call, something like `log!("TODO: vibration (AudioServicesPlaySystemSound)");`? That way the user at least knows the game _should_ have vibrated.

Reply:
Done